manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -19,6 +19,7 @@ properties:
     host_key_fingerprint: (( grab secrets.ssh_proxy_host_key.public_fingerprint ))
 
   support_address: "https://docs.cloud.service.gov.uk"
+  start_using_paas_url: "https://www.cloud.service.gov.uk/next-steps?success"
```

Comment:
I'm not particularly keen on adding this property that's not actually used by any releases. Is there a reason to define it here instead of inline below?

Reply:
No real reason. I thought We'd like to have an ability to change it quickly without digging into the manifest. I can change it if you like.

Reply:
I thought it may be OK, as the `support_address` looks similar.